Macky.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+    <!-- Bootstrap -->
+    <link href="css/bootstrap.min.css" rel="stylesheet" />
+    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
+    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
+    <!--[if lt IE 9]>
+      <script src="js/html5shiv.min.js"></script>
+      <script src="js/respond.min.js"></script>
+    <![endif]-->
+
+    <link href="css/styles.css" rel="stylesheet" />
+    <title>Macky DIARRA - technician intern</title>
+  </head>
+  <body>
+    <div id="content-head"></div>
+    <div class="container-fluid fond fond5 parallax">
+      <div class="container">
+        <div class="col-md-10 col-md-push-1 fleche-retour">
+          <a href="the-team.html"
+            ><img class="fleche-gauche" src="img/fleche-gauche.svg" /><span
+              >The team</span
+            ></a
+          >
+        </div>
+      </div>
+    </div>
+    <div class="container">
+      <div class="col-md-8 col-md-push-2">
+        <div class="fiche">
+          <div class="text-center">
+            <img
+              src="img/avatar/Macky.jpg"
+              srcset="img/avatar/Macky@2x.jpg 2x"
+              class="img-circle avatar"
+            />
+            <h3>
+              <strong>Macky DIARRA</strong><br />
+              technician intern
+            </h3>
+            <div class="reseau">
+                <a target="_blank" href="https://github.com/atrawog"><img class="twi-gith" src="img/github.svg"></a>
+            </div>
+            <p>@MackyDIARRA</p>
+          </div>
+          <div class="description">
+            <p>
+              <p>Macky DIARRA is a technician intern at QuantStack. He is currently in engineering school and is doing a general course in electrical engineering and computer engineering. He had to do some basic languages like Python, C ++, or Java.</p>
+
+              <p>Currently, he is working on the realization of an electricitymap which would allow to visualize the consumption in electricity and carbon dioxide of the terrestrial globe via dashboards realized by the creation of API keys which will allow to retrieve the data of several countries. <a target="_blank" href="https://github.com/mamba-org/quetz">Quetz</a> and making the <a target="_blank" href="https://jupyter.org">Jupyter ecosystem</a> more accessible for everyone.</p>
```

Comment:
Maybe the quetz part was a leftover from a copy paster?

```suggestion
              <p>Currently, he is working on the realization of an electricitymap which would allow to visualize the consumption in electricity and carbon dioxide of the terrestrial globe via dashboards realized by the creation of API keys which will allow to retrieve the data of several countries.
```